addon/templates/components/bootstrap/control-wrapper.hbs

```diff
@@ -3,6 +3,7 @@
     label=(component 'bootstrap/-label'
       label=label
       srOnly=srOnly
+      for=inputId
```

Comment:
This won't work for the power selects, they use aria labeled by, which doesn't use `for`

Reply:
This needs defined inside of each component as a result.

Reply:
See https://github.com/wildland/ember-bootstrap-controls/blob/master/addon/templates/components/bootstrap-power-select.hbs for an example